### ISSUE TITLE
Next simulation

### ISSUE THREAD

**Issue by sbenthall** [sbenthall/SHARKFin#246]
This will be
 - finer than the last one
 - reduce the population buying power a little.
 - will have Expectations sim_stats in the output

---------------------------

Grid - Variation:

Seeds: 20
(SHARKFin -d)

AMMPS parameters:

mm_lucas_factor =[0.0, 0.05, 0.15, 1.0]
inst_val_std = [0.1,0.15, 0.2]

SHARKFin parameters:

--attention [0.005, 0.015, 0.05]
--dphm [1000, 2178, 7389] # these should now scale with the aNrmInitMean
--zeta [0, 0.3, .7, 1.0]
--pop_aNrmInitMean [4, 5, 6, 7]

**Comment by mesalas** [comment]
Add '--mm-size' [105, 10000] to the simulation grid for the ammps config.

Per our email discussion it seems that large amount of trading by consumers may drive price away from Lucas even when market makers use the Lucas asset price.

**Comment by wjt5121** [comment]
@sbenthall @mesalas 
I've ran a single seed of the simulation grid with the mm-size parameter added. Very stable results. 
Total number of simulation is 2592 each with 4 quarters of 60 days for a total of 622080.
2441 simulations exiting with status 0 and completed all 240 days successfully for a total of 585840
149 simulations exited with status code 1. Simulations existing with code 1 completed 24104 days
2 simulations exited with status code 3. Simulations existing with code 3 completed 399 days
The total number of simulated days is 610343 which is 98.11326517489712 percent of the total.

 I've attached the results here. 
[sparksharkrun6parametergridRESULTS.csv](https://github.com/sbenthall/SHARKFin/files/11939996/sparksharkrun6parametergridRESULTS.csv)

**Comment by wjt5121** [comment]
Configuring the full 20 Seed parameter grid now. Should consist of 69120 sims.

**Comment by sbenthall** [comment]
Thank you @wjt5121 !

**Comment by wjt5121** [comment]
Crap! When I rotated the secrets I broke the connection for the currently running experiment. The scripts are not hard coded in the experiment code but look to the key vault for the connection string. When I rotated the storage key, I needed to update secret in the Key Vault as well. 
We were about 80% complete and since the the each task checks the status, I will be able to restart pending sims by simply re-executing the slurm job again

I've attached the current results (zipped csv file)  and will update with the completed results asap. 

Total number of simulation is 69120.
54829 simulations completed
13380 simulations are still pending.
79.32% complete.
[sparkshark20seedpartialrunparametergridRESULTS.csv.zip](https://github.com/sbenthall/SHARKFin/files/11960923/sparkshark20seedpartialrunparametergridRESULTS.csv.zip)

Total number of simulation is 69120 each with 4 quarters of 60 days for a total of 16588800.
53336 simulations exiting with status 0 and completed all 240 days successfully for a total of 12800640.0
1339 simulations exited with status code 1. Simulations existing with code 1 completed 234744.0 days
65 simulations exited with status code 3. Simulations existing with code 3 completed 8720.0 days
The total number of simulated days is 13044104.0 which is 78.63199266975307 percent of the total.
